### DISCUSSION TITLE
Comments in tasks

### DISCUSSION THREAD

**Discussion by omenos** [alainm23/planify#1312] [Ideas]
Similar to other platforms offering task management workflows, it would be handy to have the ability to author comment chains on a given task. It can help provide a historical record of things rather than dumping all info into the description field.

Todoist supports comments in the web app, are those sync-capable via their API?

Examples of other platforms:
- Trello
- Jira
- GitHub Projects (in a sense)
- Taiga

**Comment by stefanauss** [comment]
Todoist does indeed [support comment operations](https://developer.todoist.com/rest/v2/#comments) through their API.
